index.js

```diff
@@ -172,9 +172,13 @@ exports.stringify = function (obj, opts) {
 
 	opts = objectAssign(defaults, opts);
 
+	if (opts.sort === false) {
+		opts.sort = function () {};
```

Comment:
I'm not so happy about this mutation, but well... if you have a better idea, then I'm open to change it.